onnxruntime/core/providers/cpu/math/gemm.cc

```diff
@@ -72,6 +72,12 @@
     double,
     KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<double>()),
     Gemm<double>);
+ONNX_CPU_OPERATOR_TYPED_KERNEL(
+    Gemm,
+    13,
+    MLFloat16,
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<MLFloat16>()),
+    Gemm<MLFloat16>);
 
```

Comment:
We may also need to add support for version 7, 9 and 11

Reply:
Done.